v19.2/verify-constraint-conformance.md

```diff
@@ -0,0 +1,565 @@
+| at_risk_ranges | [`INT8`](int.html)      | The [ranges](architecture/overview.html#architecture-range) that are at risk of data loss as of the time of the report. |
+
+{{site.data.alerts.callout_info}}
+If you have not [defined any localities](configure-replication-zones.html#zone-config-node-locality), this report will not return any results. It only reports on localities that have been explicitly defined.
+{{site.data.alerts.end}}
+
+### system.replication_constraint_stats
+
+The `system.replication_constraint_stats` table lists violations to any data placement requirements you've configured, along with information about whether the cluster is actively recovering from those violations.
+
+For an example using this table, see [Find out which of your tables have a constraint violation](#find-out-which-of-your-tables-have-a-constraint-violation).
+
+{% include copy-clipboard.html %}
+~~~ sql
+SHOW COLUMNS FROM system.replication_constraint_stats;
+~~~
+
+| Column name      | Data type                     | Description                                                                                             |
+|------------------+-------------------------------+---------------------------------------------------------------------------------------------------------|
+| zone_id          | [`INT8`](int.html)            | The ID of the [replication zone](configure-zone.html).                                                  |
+| subzone_id       | [`INT8`](int.html)            | The ID of the subzone (i.e., [partition](partition-by.html)).                                           |
+| type             | [`STRING`](string.html)       | The type of zone configuration that was violated, e.g., `constraint`.                                   |
+| config           | [`STRING`](string.html)       | The YAML key-value pair used to configure the zone, e.g., `+region=europe-west1`.                       |
+| report_id        | [`INT8`](int.html)            | The ID of the [report](#system-reports_meta) that generated this data.                                  |
+| violation_start  | [`TIMESTAMP`](timestamp.html) | The time when the violation occurred, in UTC.                                                           |
+| violating_ranges | [`INT8`](int.html)            | The [ranges](architecture/overview.html#architecture-range) that are in violation of the configuration. |
+
+### system.reports_meta
+
+{% include copy-clipboard.html %}
+~~~ sql
+SHOW COLUMNS FROM system.reports_meta;
+~~~
+
+| Column name | Data type                     | Description                                                                                      |
+|-------------+-------------------------------+--------------------------------------------------------------------------------------------------|
+| id          | [`INT8`](int.html)            | The ID of the report run that generated the data available in the `system.replication_*` tables. |
```

Comment:
again s/ID of the report run that generated the data/ID of the report that this entry is part of